### PR TITLE
Automatic cleanup!

### PR DESCRIPTION
**Pull request by e111077**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
-      - google-chrome
-    packages:
-      - google-chrome-stable
-  sauce_connect: true
+  chrome: stable
 script:
   - xvfb-run polymer test
-  - >-
+  - |-
+    >-
```
Comment (bicknellr) [inline .travis.yml]: The `>-` is now passed to the shell:
https://travis-ci.org/PolymerElements/iron-checked-element-behavior/builds/382468360#L1108
I don't know enough about YAML to recommend a fix though. :/ What was this change meant to do?

Comment (e111077) [reply .travis.yml]: I don't know, I just ran what was default on the travis pass on tedium. This may be a question for @rictic or @azakus

Comment (bicknellr) [reply .travis.yml]: Here's a fun site I just found: https://yaml-multiline.info/
It looks like `|-` is "keep all the newlines, except for trailing ones" and the `>-` is "remove all the newlines". I think this means that `>-` can be removed. I'm going to try pushing some changes on top of this branch to see what happens.

**Comment by e111077** [comment]
I'm actually working on doing another tedium run. Thanks for finding a fix I can apply to the next run

**Comment by bicknellr** [comment]
Ok, I just rolled the branch back to your last commit; mine didn't work the way I thought it would.

**Comment by bicknellr** [comment]
Based on what happened with the last run, I think this might need to stay as `>-`? It tried to run all of it as one line and then dumped a bunch of code. Either way, we only need one of these.

File: .travis.yml [hunk 2]
```diff
@@ -13,21 +13,23 @@ before_script:
...
     if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer test -s 'default';
     fi
 dist: trusty
+install:
+  - npm install -g polymer-cli
+  - polymer install --variants
```
Comment (bicknellr) [inline .travis.yml]: The steps in this `install` section are at the beginning of the `before_script` section too; I'm betting one should be removed?

Comment (e111077) [reply .travis.yml]: You are correct. If I remove the install everything works. I think if you define what is in install it doesn't install anything else. e.g. the TS generator. I'm working on another run now

**Review by bicknellr — APPROVED** [review]

**Comment by bicknellr** [comment]
Since I added code here too, I'm not going to click merge myself so that you can double check me.

**Comment by bicknellr** [comment]
(Same for the remainder of the PRs in this set.)

**Comment by e111077** [comment]
LGTM

**Pull request merged**